server/oauth2.go

```diff
@@ -20,6 +20,7 @@ import (
 	"time"
 
 	jose "gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2/jwt"
```

Comment:
we don't need to use the jwt package for this. The jose.ParseSigned works fine https://godoc.org/gopkg.in/square/go-jose.v2#ParseSigned